profiling/src/lib.rs

```diff
@@ -516,6 +516,8 @@ extern "C" fn rinit(_type: c_int, _module_number: c_int) -> ZendResult {
                 // standardized tag name.
                 add_tag(&mut tags, "runtime_version", PHP_VERSION.as_str());
                 add_tag(&mut tags, "php.sapi", SAPI.as_ref());
+                let runtime_engine = if cfg!(php_zts) { "ZTS" } else { "NTS" };
+                add_tag(&mut tags, "runtime_engine", runtime_engine);
```

Comment:
I suspect we may want to do this instead: `nts-ndebug`, `nts-debug`, `zts-ndebug`, `zts-debug`. Thought?